docker-compose.yaml

```diff
@@ -42,9 +42,9 @@ services:
       # First line overrides an existing .env, if any.
       # This is to make sure that it is clean even though docker volume was not cleaned.
       - |
-        echo "KAKAROT_ADDRESS=$(jq -r '.kakarot.address' /deployments/deployments.json)" > /deployments/.env;
-        echo "DEPLOYER_ACCOUNT_ADDRESS=$(jq -r '.deployer_account.address' /deployments/deployments.json)" >> /deployments/.env;
-        echo "PROXY_ACCOUNT_CLASS_HASH=$(jq -r '.proxy' /deployments/declarations.json)" >> /deployments/.env
+        echo "KAKAROT_ADDRESS=$(jq -r '.kakarot.address' /deployments/custom_rpc/deployments.json)" > /deployments/.env;
+        echo "DEPLOYER_ACCOUNT_ADDRESS=$(jq -r '.deployer_account.address' /deployments/custom_rpc/deployments.json)" >> /deployments/.env;
+        echo "PROXY_ACCOUNT_CLASS_HASH=$(jq -r '.proxy' /deployments/custom_rpc/declarations.json)" >> /deployments/.env
```

Comment:
Tested in docker locally both for katana and madara

![image](https://github.com/kkrt-labs/kakarot-rpc/assets/35031356/2e185db8-5c06-4f4b-aaef-6a10bff12df3)

Reply:
see my comment in the kakarot PR, I'd rather have `custom-rpc`